test/cypress/plugins/stubs.js

```diff
@@ -475,6 +475,31 @@ module.exports = {
       }
     ]
   },
+  patchSuccess: (opts = {}) => {
```

Comment:
This should be a stub that is only used to update the merchant name, and should use the new `validUpdateMerchantNameRequest` fixture you've added.

This will improve our confidence in the Cypress tests, as we know that we're building a request using a strict fixture that has been validated by the Pact tests. In this case, the Pact tests validate that the `path` in the request, `merchant_details/name` is a valid path as this is hardcoded into the fixture.